ui/src/plugins/dev.perfetto.QueryPage/query_page.ts

```diff
@@ -93,18 +98,68 @@ export class QueryPage implements m.ClassComponent<QueryPageAttrs> {
           }),
           m(
             Stack,
-            {orientation: 'horizontal', className: 'pf-query-page__hotkeys'},
+            {
+              orientation: 'horizontal',
+              className: 'pf-query-page__hotkeys',
+            },
             'or press',
             m(HotkeyGlyphs, {hotkey: 'Mod+Enter'}),
           ),
           m(StackAuto), // The spacer pushes the following buttons to the right.
+          attrs.trace.isInternalUser &&
+            m(Button, {
+              icon: 'star',
+              title:
+                'Generate SQL queries with the Perfetto SQL Agent! Give feedback: go/perfetto-llm-bug',
+              label: 'Generate SQL Queries with AI',
+              onclick: () => {
+                window.open('http://go/perfetto-sql-agent', '_blank');
+              },
+            }),
           m(CopyToClipboardButton, {
             textToCopy: attrs.editorText,
             title: 'Copy query to clipboard',
             label: 'Copy Query',
           }),
         ]),
       ]),
+      this.shouldDisplayPerfettoSqlAgentBanner(attrs) &&
+        m(
+          Box,
+          m(
+            Callout,
+            {
+              icon: 'star',
```

Comment:
I just updated the material icons pack to the latest version, which includes a bunch more icons, if you wanted to use the [wand_stars](https://fonts.google.com/icons?selected=Material+Symbols+Outlined:wand_stars:FILL@0;wght@400;GRAD@0;opsz@24&icon.query=magic&icon.size=24&icon.color=%23e8eaed) icon or similar here?